observability/network_observability/network-observability-overview.adoc

```diff
@@ -22,7 +22,15 @@ The Network Observability Operator provides the Flow Collector API custom resour
 [id="no-console-integration"]
 == {product-title} console integration
 
-{product-title} console integration offers overview, topology view and traffic flow tables.
+{product-title} console integration offers overview, topology view and traffic flow tables in both *Administrator* and *Developer* perspectives.
+
+In the *Administrator* perspective, the Network Observability *Overview*, *Traffic flows* and *Topology* views can be found by clicking *Observe* -> *Network Traffic*. In the *Developer* perspective, you can view this information by clicking *Observe*. The Network Observability metrics dashboards in *Observe* -> *Dashboards* are only available to administrators. 
+
+[NOTE]
+====
+To enable multi-tenancy for the developer perspective and for administrators with limited access to namespaces, you must specify permissions by defining roles. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-multi-tenancy_network_observability[Enabling multi-tenancy in Network Observability]
+====
+to you must specify roles 
```

Comment:
Think L33 should be removed?

Reply:
Oh yep, looks like a copy/paste holdover. Thanks for catching that.